<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QLabel" name="age_fish">
      <property name="text">
       <string>Age of fish</string>
      </property>
      <property name="buddy">
       <cstring>age_fishBox</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QSpinBox" name="age_fishBox">
      <property name="minimum">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QListWidget" name="palette">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QSpinBox" name="age_sharkBox">
      <property name="minimum">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="5">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>749</width>
         <height>754</height>
        </rect>
       </property>
       <zorder>simulation</zorder>
      </widget>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="age_shark">
      <property name="text">
       <string>Age of shark</string>
      </property>
      <property name="buddy">
       <cstring>age_sharkBox</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="energy_initial">
      <property name="text">
       <string>Initial energy </string>
      </property>
      <property name="buddy">
       <cstring>energy_initialBox</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QSpinBox" name="energy_initialBox">
      <property name="minimum">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="energy_eat">
      <property name="text">
       <string>Energy after eating</string>
      </property>
      <property name="buddy">
       <cstring>energy_eatBox</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QSpinBox" name="energy_eatBox">
      <property name="minimum">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionNext_chronon"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNext_chronon"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNext_chronon">
   <property name="text">
    <string>Next chronon</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>age_fishBox</tabstop>
  <tabstop>age_sharkBox</tabstop>
  <tabstop>energy_initialBox</tabstop>
  <tabstop>energy_eatBox</tabstop>
  <tabstop>palette</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
